Making block_devices non mandatory in EC2 block definition.

lukaszraczylo opened the pull request:
This fixes #3

lukaszraczylo committed:
Making block_devices non mandatory in EC2 block definition.

This fixes #3

Fulfills following acceptance criteria:

- Sane defaults provided for block_devices section
If no block devices specified bootstrap-cfn will create /dev/sda1 with 20GB of space
as a default.

- Tests added to cover this case
- Docs added to readme.

ashb commented on tests/tests.py:
```diff
@@ -542,6 +542,13 @@ def test_ec2(self):
+        project_config.config.pop('rds')
+        project_config.config['ec2'].pop('block_devices')
+        config = ConfigParser(project_config.config, 'my-stack-name')
+        config.process()
```
In this case the behavior is entirely in the `ec2()` function so could you change the test to 1) call that, and 2) check the return value (or at least the devices section) is defaulted appropriately.

lukaszraczylo committed:
Test fix - checking output only for EC2, compare with expected.

From now on - it checks returned value and compares it with expected
/dev/sda1:20G ( which is different than 10G from tests/sample-project.yml )
